### ISSUE TITLE
Some kind of online help

### ISSUE THREAD

**Issue by Eumaeus** [cite-architecture/CITE-App#46]
My inclination is to do "help" in the app's GitHub Wiki, and just include a link to that on the app's page. That way, you, I, our students, Neven, or anyone can build out help as needed. 

Is this a good idea, or a terrible idea?

**Comment by neelsmith** [comment]
Maybe slightly better variant:  let's get up a ghpages web site with some help we author, links to APIs, and *link* to a wiki anyone can contribute to.  Both what you suggest AND our own stuff if we choose to.

**Comment by neelsmith** [comment]
(But definitely keep help functions outside the app!)

**Comment by Eumaeus** [comment]
I've started a basic wiki. I despise working with ghpages and that damned Jekyll. But of course it is best. I'll build out some help in the Wiki, for now, and we can move it to ghpages at some point in the future.

**Comment by neelsmith** [comment]
We don't need to do ghpages if you hate it.  It's just a reliable, free web host you can throw markdown at, so I've used it for that reason.

Still, I'm glad to throw some stuff together on ghpages if you think we should

**Comment by Eumaeus** [comment]
No, let's use ghpages. They look good and are easy to find and professional-seeming. Once it is all set up It is easy enough to add stuff.
I
